file_optical.c

```diff
@@ -54,6 +54,7 @@
 #include "tcmu-runner.h"
 #include "tcmur_device.h"
 #include "libtcmu.h"
+#include "libtcmu_common.h"
```

Comment:
To include the headers wherever needed is always a good idea.

But for this the `libtcmu_common.h` has been already included by `tcmu-runner.h` and `libtcmu.h`, this fixing here actually will do nothing. I think this `undefined symbol` error should be caused you were using the old version of `handler_file_optical.so` ?